kg_covid_19/transform_utils/scibite_cord/scibite_cord.py

```diff
@@ -263,6 +284,125 @@ def extract_termite_hits(self, data: Dict) -> Set:
                     self.concept_name_map[hit['id']] = hit['name']
         return terms
 
+    def contract_uri(self, iri) -> str:
+        """Contract a given IRI.
+
+        Contract a given IRI, with special parsing and transformations
+        depending on the nature of the IRI.
+
+        Args:
+            iri: IRI as string
+
+        Returns:
+            str.
+
+        """
+        curie = ""
+        if 'http://www.genenames.org/cgi-bin/gene_symbol_report?match=' in iri:
+            identifier = iri.split('=')[-1]
+            if identifier in self.gene_info_map:
+                curie = f"NCBIGene:{self.gene_info_map[identifier]['NCBI']}"
+        else:
+            if self.is_iri(iri):
+                curie = contract_uri(iri)
+                if curie:
+                    curie = curie[0]
+                else:
+                    curie = contract_uri(iri, cmaps=[CUSTOM_CMAP])
+                    print(f"Contracting URI {iri} to {curie} via custom map")
```

Comment:
One more request - could we omit or making this into a `logging.DEBUG()` call instead? I'm running the ingest and it's a little chatty